assets/docs/rishit-dagli/swin-transformers/1.md

```diff
@@ -0,0 +1,35 @@
+# Module rishit-dagli/swin-transformers/1
```

Comment:
Can we change the name to the singular `swin-transformer` (and rename the parent directory accordingly)? Since it only is one model, I think the singular name would be more appropriate. In case more of these are published in the future (PUBLISHER/swin-transformer-xl/1, ...), one could create a collection at e.g. tfhub.dev/rishit-dagli/collections/swin-transformers/1 to reference all the swin transformers that have already been published.

Reply:
Got it, made the change